pkgs/applications/networking/feedreaders/feeds/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+
+, appstream
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+
+, glib
+, gtk3
+, libhandy
+, webkitgtk
+, python3
+}:
+let
+  listparser = import ./listparser.nix {
```

Comment:
I think now it's impossible to override it in case it fails to build.. You should also be able to use `callPackage` instead which is a bit more versatile - you won't need `inherit lib python3;`. If we would like to not add it to `python-packages.nix`, we can put in the arguments list:

```nix
, listparser ? callPackage ./listparser.nix { }
```

And then inside the feeds expression, use `passthru = { inherit listparser; };` - this will expose it and allow it to be overridden in case it fails to build.

Reply:
Can you explain why it should go into `passthru`? I don't quite see how that's going to help with overriding.

Reply:
With the passthru, one should be able to:

```nix
self: super: {
  feeds-with-different-listparser = super.feeds.override {
    # No need to use the `passthru` attribute, but it helps the usage be clear
    listparser = super.feeds.passthru.listparser.overrideAttrs(oldAttrs: {
      /* src = ...; */
    });
  };
}
```